src/components/Tool/Appeal/Modals/PledgeModal/PledgeModal.test.tsx

```diff
@@ -141,7 +141,7 @@ describe('PledgeModal', () => {
     expect(getByText('Currency')).toBeInTheDocument();
 
     expect(
-      await findByRole('combobox', { name: 'Currency' }),
```

Comment:
This test needing to change is a sign that we accidentally broke the labels in the PR. If you add back `id="amountCurrency-select"` to the autocomplete, it should work. Then you can switch this back to `getByRole`. Make sure to remove the `data-testid` too because it's not needed anymore.